object-construction-checker/src/main/java/org/checkerframework/checker/objectconstruction/framework/LombokSupport.java

```diff
@@ -116,7 +116,9 @@ private void handleToBuilderType(
     List<String> requiredProperties = getLombokRequiredProperties(classElement);
     AnnotationMirror calledMethodsAnno =
         atypeFactory.createCalledMethods(requiredProperties.toArray(new String[0]));
-    type.replaceAnnotation(calledMethodsAnno);
+    if (!type.isAnnotatedInHierarchy(calledMethodsAnno)) {
+      type.addAnnotation(calledMethodsAnno);
+    }
```

Comment:
What is the purpose of this change? It seems unrelated, and I would expect it to cause tests to fail.

Reply:
This was related to the changes in the Returns Receiver Checker but now it seems unnecessary, and restored to the original version.